browser/registry.go

```diff
@@ -199,12 +199,18 @@ func newBrowserRegistry(vu k6modules.VU, remote *remoteRegistry, pids *pidRegist
 		buildFn: builder,
 	}
 
+	exitSubID, exitCh := vu.Events().Global.Subscribe(
+		k6event.Exit,
```

Comment:
Will each `vu` get an `Exit` event when emitted?

Reply:
Yes, as every VU subscribes to the exit event, every VU browser registry will get an `Exit` event so it can do the cleanup.